cellpack/autopack/__init__.py

```diff
@@ -387,6 +387,10 @@ def load_file(filename, destination="", cache="geometries", force=None):
         if database_name == "firebase":
             db = DATABASE_IDS.handlers().get(database_name)
             initialize_db = db()
+            if not initialize_db._initialized:
+                sys.exit(
+                    "The selected database is not initialized. Please set up Firebase credentials to pack remote recipes."

```

Comment:
Could link to the readme/instructions here similar to how we do for missing AWS credentials